files.js

```diff
@@ -13,29 +13,27 @@ function saveTypedArrayFile(filename, seq, count, tarr, cb) {
   b.writeInt32LE(count, 4)
   dataBuffer.copy(b, 8)
 
-  const f = AtomicFile(filename)
-  f.set(b, cb)
+  writeFile(filename, b).then(cb).catch(cb)
```

Comment:
Are you sure this promise resolves with undefined or null? If it resolves with anything else, then it would do `.then(x => cb(x))` which means delivering `x` as an error, which is probably not what we want

Reply:
Good spotting. I did check that in both node and browser this returns undefined, but in order to be sure that this will always work I made a small change so it will never return anything. Thanks